src/analyzer/protocol/ssl/CMakeLists.txt

```diff
@@ -10,8 +10,8 @@ zeek_plugin_bif(events.bif)
 zeek_plugin_bif(functions.bif)
 zeek_plugin_bif(consts.bif)
 zeek_plugin_pac(tls-handshake.pac tls-handshake-protocol.pac tls-handshake-analyzer.pac ssl-defs.pac
-	proc-client-hello.pac
-	proc-server-hello.pac
+	proc-client-hello-tls.pac
```

Comment:
I assume I am lazy here - is it absolutely necessary to split these up?

Reply:
The only diff in the `-tls` versions is, that we store `client_random` and `server_random` in the connection which only applies for TLS handshake connections but not SSL connections. I assumed that the files only existed to not duplicate the identical methods between SSL and TLS. However, as they are now not identical anymore, we could also just move the definitions into the respective `*-analyzer.pac` files and remove the extra files.

Reply:
Yup, that would probably make more sense. Especially since we then also can throw out some of the logic that checks if this was an SSL or a TLS event.

Reply:
Done, although I did not find any logic related to SSL/TLS checks directly in the files. They might be somewhere else though?